core/cmd/shell_remote.go

```diff
@@ -511,6 +512,23 @@ func (s *Shell) checkRemoteBuildCompatibility(lggr logger.Logger, onlyWarn bool,
 	return nil
 }
 
+func (s *Shell) Health(c *cli.Context) error {
+	mime := gin.MIMEPlain
+	if c.Bool("json") {
+		mime = gin.MIMEJSON
+	}
+	resp, err := s.HTTP.Get("/health", map[string]string{"Accept": mime})
+	if err != nil {
+		return s.errorOut(err)
+	}
+	b, err := parseResponse(resp)
+	if err != nil {
+		return s.errorOut(err)
+	}
+	fmt.Println(string(b))
```

Comment:
is this print intentional? No formatting?

Reply:
Correct. No formatting necessary. Either it is already plaintext, or they requested raw JSON via the flag anyways. We could pretty print, but the text version is already the human formatted version of the JSON, and it is trivial for them to use `| jq` if they really need that anyways.